docs/config/apache.md

```diff
@@ -16,24 +16,24 @@ description: Add an Apache webserver service to your Lando config for local deve
 ## Patch versions
 
 ::: warning Not officially supported!
-While we allow users to specify patch versions for this service they are not *officially* supported so if you use one YMMV.
+While we allow users to specify patch versions for this service, they are not *officially* supported, so if you use one, YMMV.
 :::
 
-To use a patch version you can do something like this:
+To use a patch version, you can do something as shown below:
 
 ```yaml
 services:
   my-service:
     type: apache:2.4.33
 ```
 
-But make sure you use one of the available [patch tags](https://hub.docker.com/r/bitnami/apache/tags) for the underlying image we are using.
+But make sure you use one of the available [patch tags](https://hub.docker.com/r/bitnami/apache/tags) for our underlying image.
```

Comment:
Our is not accurate. Most the images are not "our" images and its important we do not communicate that they are.

Reply:
Thanks for the feedback.  Fixed.